code/game/objects/items/storage/bags.dm

```diff
@@ -346,6 +346,26 @@
 	STR.insert_preposition = "in"
 	STR.can_hold = typecacheof(list(/obj/item/reagent_containers/pill, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/medspray, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/dropper))
 
+/*
+ *	Grenade bag
+ */
+
+/obj/item/storage/bag/grenade_parts
+	name = "grenade maker's bag"
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "bag_grenade"
+	desc = "A bag for storing grenade components."
+	w_class = WEIGHT_CLASS_TINY
+	resistance_flags = FLAMMABLE
+
+/obj/item/storage/bag/grenade_parts/ComponentInitialize()
+	. = ..()
+	GET_COMPONENT(STR, /datum/component/storage)
+	STR.max_combined_w_class = 200

```

Comment:
will you ever reach this number?

Reply:
I don't actually know what that variable does, but it's the same as the Chemistry bag.

Reply:
it's the sum of item's weights.

I think all assemblies listed are tiny (1) so at most you could ever get is 50 which is why I asked.